### DISCUSSION TITLE
generate spec: how to overide array type

### DISCUSSION THREAD

**Discussion by Luap99** [go-swagger/go-swagger#2770] [Q&A]
I have a struct like this
```go
type Options struct {
	// StaticIPs for this container. Optional.
	StaticIPs []net.IP `json:"static_ips,omitempty"`
}
```

Currently this generates a `Array of Array of integers`. However due the custom Marshal/Unmarshal functions it is actually a array of strings in the final json object.

I know that I can force a single element to string with `swagger:strfmt` but this this case I want it to be an array of strings. How can I achieve this?

**Comment by rhatdan** [comment]
friendly ping

**Comment by casualjim** [comment]
I think this might work:

https://github.com/go-swagger/go-swagger/blob/master/fixtures/goparsing/classification/models/extranomodel.go#L102

**Reply by Luap99** [reply]
Unfortunately this does not work, `// swagger:type` seems to get ignored. Only `// swagger:strfmt` is respected.
